internal/adapters/dataproviders/liquidity_provider.go

```diff
@@ -189,27 +189,6 @@ func (lp *LocalLiquidityProvider) PeginConfiguration(ctx context.Context) liquid
 	return configuration.Value
 }
 
-func validateConfiguration[T liquidity_provider.ConfigurationType](
-	displayName string,
-	lp *LocalLiquidityProvider,
-	readFunction func() (*entities.Signed[T], error),
-) (*entities.Signed[T], error) {
-	configuration, err := readFunction()
-	if err != nil {
-		log.Errorf("Error getting %s configuration, using default configuration. Error: %v", displayName, err)
-		return nil, err
-	}
-	if configuration == nil {
-		log.Warnf("Custom %s configuration not found. Using default configuration.", displayName)
-		return nil, errors.New("configuration not found")
-	}
-	if err = configuration.CheckIntegrity(crypto.Keccak256); err != nil {
```

Comment:
here was ok to use crypto package because is not the entity layer but now that it was moved it should be received by parameter

Reply:
Noted.